vcpkg-overlays/dynamic-libssl3/portfile.cmake

```diff
@@ -0,0 +1 @@
+set(VCPKG_POLICY_EMPTY_PACKAGE enabled)
```

Comment:
I think the overlay should instead be defined in/used by the cpack builds (for distros that provide openssl 3) under `.github/actions/openziti-tunnel-build-action/(redhat|ubuntu*)`. This way the grab-and-go binaries continue to be uniquely identified by the target architecture and nothing else.

Yes, this would mean that you need to use the repos to get a fips-ready binary, but I think that's fair. Keep in mind that we're still (trying to) support ubuntu 18 with the gh-release binaries, and ubuntu 18 does not provide libssl3.

Reply:
You're right. It makes more sense to take this opportunity to switch the Jammy CPack builder from static to dynamic, since we're eliminating the dependency on the newer version of libssl3 that's unavailable from Jammy's repos.

Reply:
I plan to mark this ready for review after incorporating Eugene's patch to eliminate the dependency on newer libssl3 and verifying the DEB produced by the updated Jammy builder has the expected libssl3 links.

Meanwhile, I welcome further input on the draft PR.